React/Views/RCTScrollView.m

```diff
@@ -317,8 +317,8 @@ - (void)setFrame:(CGRect)frame
   CGSize boundsSize = self.bounds.size;
 
   self.contentOffset = CGPointMake(
-    MAX(0, MIN(originalOffset.x, fullContentSize.width - boundsSize.width)),
-    MAX(0, MIN(originalOffset.y, fullContentSize.height - boundsSize.height)));
+    MAX(-contentInset.left, MIN(originalOffset.x, fullContentSize.width - boundsSize.width)),
```

Comment:
What the point of this change? I always thought that `contentOffset` already "contains" `contentInset`, so when `contentOffset` is `{0, 0}`, the insets are visible. So, when `contentOffset` is `{-contentInset.top, 0}` we have doubled top inset. Am I wrong?